### DISCUSSION TITLE
How to add IDE support for the auto-imports ?

### DISCUSSION THREAD

**Discussion by Davounet** [nuxt/nuxt#17763] [Questions] [answered]
Hello !

I am migrating a typescript nuxt2 project to nuxt-bridge, the auto-imports (`ref`, `computed`, `useRouter`...) are working fine but are not resolved by my IDE (VSCode with Vetur/Volar) which hits me with a lot of those squiggly red underlines.
Is there something specific to do to fix this situation ?

Thank you very much, have a nice day !

**Comment by danielroe** [comment, marked as answer]
Have you done this? https://v3.nuxtjs.org/getting-started/bridge#update-tsconfigjson

**Reply by Davounet** [reply]
I did but the problem was there indeed ! By switching branches the `.nuxt` directory was reset to the nuxt2 version, without the `tsconfig`file, which could not be imported.
Thank you very much !
